docs/configuration/configuration.md

```diff
@@ -66,6 +66,10 @@ global:
   external_labels:
     [ <labelname>: <labelvalue> ... ]
 
+  # File to which PromQL queries are logged.
+  # Reloading the configuration will reopen the file.
+  [ query_log_file: <string> ]
```

Comment:
If I recall correctly we discussed at the devsummit that we should document how to allow people who want to to have this be part of the stdout log stream and have some docs around it. I assume that it's just a matter of setting this to `/dev/stdout`, I just wonder about stream syncronization.

Reply:
Probably best for other docs. Does `CON` still exist on Windows?

Reply:
Is it worth a complete section? under Configuration? under Querying?

![op](https://user-images.githubusercontent.com/291750/72051473-a9fa8300-32c3-11ea-8531-69d656708375.png)

Reply:
Querying is for the PromQL language.

It wouldn't be part of the reference docs at all, this is generic Unix/Windows knowledge.

Reply:
Should it be a small post on the blog, with example with logrotate etc ?

Reply:
(blog post would be published after the release)

Reply:
I think a blog with examples for logrotate and stdout sounds like a great idea, or a user guide.

Reply:
I think a blog with examples for logrotate and stdout sounds like a great idea, or a user guide.